api/payara-bom/pom.xml

```diff
@@ -419,7 +419,7 @@
             <dependency>
                 <groupId>org.apache.santuario</groupId>
                 <artifactId>xmlsec</artifactId>
-                <version>${org.apache.santuario.version}</version>
+                <version>${xmlsec.version}</version>
```

Comment:
By my suggested "use project names" convention this should probably be `santuario.version`